mwdb/web/src/components/RecentView/RecentView.js

```diff
@@ -54,6 +54,8 @@ export default function RecentView(props) {
     // Query error shown under the query bar
     let [queryError, setQueryError] = useState(null);
 
+    let [objectCount, setObjectCount] = useState(null)
```

Comment:
Maybe it would be nice to reset the object count along with errors (`setObjectCount(null)` in `resetErrors`)